middle_end/flambda2/simplify/env/downwards_env.ml

```diff
@@ -95,61 +95,64 @@ type t =
-      @[<hov 1>(cost_of_lifting_continuation_out_of_current_one %d)@]\
-      )@]"
-    round
-    Target_system.Machine_width.print machine_width
-    TE.print typing_env
-    Inlined_debuginfo.print inlined_debuginfo
-    Disable_inlining.print disable_inlining
-    Inlining_state.print inlining_state
-    propagating_float_consts
-    at_unit_toplevel
-    Continuation.print unit_toplevel_return_continuation
-    Continuation.print unit_toplevel_exn_continuation
-    Variable.Set.print variables_defined_at_toplevel
-    CSE.print cse
-    (Variable.Map.print Comparison_result.print) comparison_results
-    Are_rebuilding_terms.print are_rebuilding_terms
-    Closure_info.print closure_info
-    (Code_id.Map.print Code.print) all_code
-    Loopify_state.print loopify_state
-    Replay_history.print replay_history
-    Specialization_cost.print specialization_cost
-    (Format.pp_print_list ~pp_sep:Format.pp_print_space Lifted_cont_params.print) defined_variables_by_scope
-    cost_of_lifting_continuations_out_of_current_one
+let print ppf
+    { round;
+      machine_width;
+      typing_env;
+      inlined_debuginfo;
+      disable_inlining;
+      inlining_state;
+      propagating_float_consts;
+      at_unit_toplevel;
+      unit_toplevel_exn_continuation;
+      variables_defined_at_toplevel;
+      cse;
+      comparison_results;
+      are_rebuilding_terms;
+      closure_info;
+      unit_toplevel_return_continuation;
+      all_code;
+      get_imported_code = _;
+      inlining_history_tracker = _;
+      loopify_state;
+      replay_history;
+      specialization_cost;
+      defined_variables_by_scope;
+      lifted = _;
+      cost_of_lifting_continuations_out_of_current_one
+    } =
+  let open! Misc.Sexp in
+  print ppf
+    [ i "round" round;
+      p "machine_width" Target_system.Machine_width.print machine_width;
+      p "typing_env" TE.print typing_env;
+      p "inlined_debuginfo" Inlined_debuginfo.print inlined_debuginfo;
+      p "disable_inlining" Disable_inlining.print disable_inlining;
+      p "inlining_state" Inlining_state.print inlining_state;
+      b "propagating_float_consts" propagating_float_consts;
+      b "at_unit_toplevel" at_unit_toplevel;
+      p "unit_toplevel_return_continuation" Continuation.print
+        unit_toplevel_return_continuation;
+      p "unit_toplevel_exn_continuation" Continuation.print
+        unit_toplevel_exn_continuation;
+      p "variables_defined_at_toplevel" Variable.Set.print
+        variables_defined_at_toplevel;
+      p "cse" CSE.print cse;
+      p "comparison_results"
+        (Variable.Map.print Comparison_result.print)
+        comparison_results;
+      p "are_rebuilding_terms" Are_rebuilding_terms.print are_rebuilding_terms;
+      p "closure_info" Closure_info.print closure_info;
+      p "all_code" (Code_id.Map.print Code.print) all_code;
+      p "loopify_state" Loopify_state.print loopify_state;
+      p "binding_histories" Replay_history.print replay_history;
```

Comment:
(This is the only item whose name is
different from the string - is it on
purpose?)